VK/VK/NetworkService/NetworkServiceble.swift

```diff
@@ -0,0 +1,163 @@
+        var urlComponents = URLComponents()
+        urlComponents.scheme = QueryItems.schemeName.rawValue
+        urlComponents.host = QueryItems.apiHostName.rawValue
+        urlComponents.path = QueryItems.apiPathName.rawValue + QueryItems.friendListMethodName.rawValue
+        urlComponents.queryItems = defaultQueryItems + makeURLQueryItems(itemsMap: parametersMap)
+
+        guard let url = urlComponents.url else { return }
+        print(url)
+        sendRequest(url: url, method: .get, model: ResponseWithFriends.self) { result in
+            complition(result)
+        }
+    }
+
+    func fetchClientsGroups(parametersMap: [String: String], complition: @escaping (ResponseWithGroups) -> ()) {
+        var urlComponents = URLComponents()
+        urlComponents.scheme = QueryItems.schemeName.rawValue
+        urlComponents.host = QueryItems.apiHostName.rawValue
+        urlComponents.path = QueryItems.apiPathName.rawValue + QueryItems.groupsListMethodName.rawValue
+        urlComponents.queryItems = defaultQueryItems + makeURLQueryItems(itemsMap: parametersMap)
+
+        guard let url = urlComponents.url else { return }
+        sendRequest(url: url, method: .get, model: ResponseWithGroups.self) { result in
+            complition(result)
+        }
+    }
+
+    func fetchFoundGroups(parametrsMap: [String: String], complition: @escaping (ResponseWithGroups) -> ()) {
+        var urlComponents = URLComponents()
+        urlComponents.scheme = QueryItems.schemeName.rawValue
+        urlComponents.host = QueryItems.apiHostName.rawValue
+        urlComponents.path = QueryItems.apiPathName.rawValue + QueryItems.searchMethodName.rawValue
+        urlComponents.queryItems = defaultQueryItems + makeURLQueryItems(itemsMap: parametrsMap)
+
+        guard let url = urlComponents.url else { return }
+
+        sendRequest(url: url, method: .get, model: ResponseWithGroups.self) { result in
+            complition(result)
+        }
+    }
+
+    func fetchAllPhoto(parametersMap: [String: String], complition: @escaping (Codable) -> ()) {
+        var urlComponents = URLComponents()
+        urlComponents.scheme = QueryItems.schemeName.rawValue
+        urlComponents.host = QueryItems.apiHostName.rawValue
+        urlComponents.path = QueryItems.apiPathName.rawValue + QueryItems.getPhotoMethodName.rawValue
+        urlComponents.queryItems = defaultQueryItems + makeURLQueryItems(itemsMap: parametersMap)
+
+        guard let url = urlComponents.url else { return }
+        sendRequest(url: url, method: .get, model: ResponseWithPhoto.self) { result in
+            complition(result)
+        }
+    }
+
+    func fetchPhoto(by urlString: String, complition: @escaping (Data) -> ()) {
+        guard let url = URL(string: urlString) else { return }
+        DispatchQueue.global().async {
+            AF.request(url).responseData { response in
+                guard let data = response.data else { return }
+                complition(data)
```

Comment:
```suggestion
                completion(data)
```
включи проверку правописания